packages/babel-generator/test/index.js

```diff
@@ -1492,6 +1492,26 @@ describe("programmatic generation", function () {
   val
 );`);
     });
+
+    it("multi-line leading comment after return 2", () => {
```

Comment:
This test has nothing to do with this PR, it just ensures that https://github.com/babel/babel/pull/16104 does not cause regressions.